Gir.toml

```diff
@@ -166,3 +168,10 @@ status = "generate"
     [[object.function]]
     name = "copy"
     ignore = true
+
+[[object]]
+name = "GtkSource.StyleScheme"
+status = "generate"
+    [[object.function]]
+    name = "get_style"
+    version = "3.16"
```

Comment:
What reason to add version?
https://developer.gnome.org/gtksourceview/stable/GtkSourceStyleScheme.html#gtk-source-style-scheme-get-style don't say anything about this.

Reply:
It doesn't compile in another file otherwise (closed the terminal so lost the logs... :-/). Remove this line and regen, you'll see directly. However, if you have a better solution, I'm all for it!

Reply:
Strange, I try it at home

Reply:
Thanks!